docs/04-operation-guides/troubleshooting/apix-09-upgrade-sidecar-proxy.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Pods stuck in Pending/Failed/Unknown state after upgrade
+---
+
+## Symptom
+
+You cannot access services or functions using the API Rules created. The Kyma Gateway refuses the connection.
+Some of your Pods are stuck in the Pending/Failed/Unknown state, or their sidecar proxy version differ from the installed Istio version.
+
+## Cause
+
+During the upgrade Kyma will trigger rollout restart to instance of resources to ensure full compatibility of sidecar proxy with newly installed Istio version. Exception to this are standalone pods (without OwnerReference) or pods created by Job controller. Pods that cannot complete rollout restart process or rollout restart cannot be triggered because a pod is standalone or created by Job resource will get annotation `istio.reconciler.kyma-project.io/proxy-reset-warning` with brief explanation of the cause.
```

Comment:
```suggestion
During the upgrade, Kyma triggers rollout restart to the instance of resources, in order to ensure full compatibility of sidecar proxy with the newly installed Istio version. Exception to this are standalone Pods (without owner reference) or Pods created by the Job controller. Pods that cannot complete the rollout restart process, or the restart cannot be triggered because a Pod is a standalone resource or is created by the Job controller, get annotation `istio.reconciler.kyma-project.io/proxy-reset-warning` with brief explanation of the cause.
```
According to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/#owner-references-in-object-specifications) owner reference is not capitalized.